conf/codon.config

```diff
@@ -12,36 +12,13 @@ params {
 workDir = params.workdir
 
 profiles {
-    ebi_lsf {
-
-        includeConfig 'codon_dbs.config'
-
-        executor {
-            name = "lsf"
-            queueSize = 200
-            queueGlobalStatus = true
-            submitRateLimit = "10 sec"
-            pollInterval = "10 sec"
-        }
-
-        process {
-            queue = {
-                task.memory >= 200.GB ? 'bigmem' : 'production'
-            }
-        }
-
-        conda.enabled = false
-
-        singularity {
-            enabled = true
-            autoMounts = true
-            cacheDir = params.singularity_cachedir
```

Comment:
Because this line was removed, now variable `singularity_cachedir` is used only once. Maybe remove it and just use the str in `singularity.cacheDir` ?

Reply:
I left that one in because it could be useful to override it anyway.